pyproject.toml

```diff
@@ -15,6 +15,7 @@ jupyter = "^1.0.0"
 ipykernel = "^6.29.5"
 polars = "^1.5.0"
 pyarrow = "^17.0.0"
+azuretools = {git = "https://github.com/cdcent/cfa-stf-azuretools"}
```

Comment:
We can make this depend on some secret https://github.com/orgs/python-poetry/discussions/3794